src/mixer/playermanager.cpp

```diff
@@ -60,6 +60,30 @@ bool extractIntFromRegex(const QRegularExpression& regex, const QString& group,
     return true;
 }
 
+/// Returns the first object from a list of `BaseTrackPlayer` instances where
+/// the corresponding `play` CO is set to 0.
+template<class T>
+T* findFirstStoppedPlayerInList(const QList<T*>& players) {
+    for (T* pPlayer : players) {
```

Comment:
this could be const if `BasePlayer::getGroup()` was const (which it should be IMO).

Reply:
Do you mean this?
```suggestion
    for (const T* pPlayer : players) {
```
This does not work, because in that case the return type would also have to be `const`. This would severly limit what you can do with the result (or you'd need to use `const_cast` which I consider a terrible hack).

Reply:
Good point. I didn't think about that. LGTM me then.